Formula/python@3.9.rb

```diff
@@ -0,0 +1,335 @@
+class PythonAT39 < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "https://www.python.org/"
+  url "https://www.python.org/ftp/python/3.9.0/Python-3.9.0rc1.tar.xz"
```

Comment:
Is it worth trying release candidate 2 or is that more work than it is worth?
* https://www.python.org/download/pre-releases/

Reply:
At this point yeah we can switch to rc2 since all that's really being tested is the build of python itself.